xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DocumentsMacro.xml

```diff
@@ -793,20 +845,31 @@ Generates:
         <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
       </name>
       <type>
+        <cache>0</cache>
+        <defaultValue>Unknown</defaultValue>
         <disabled>0</disabled>
+        <displayType>input</displayType>
+        <freeText>allowed</freeText>
+        <largeStorage>1</largeStorage>
+        <multiSelect>0</multiSelect>
         <name>type</name>
         <number>5</number>
+        <picker>1</picker>
         <prettyName>Parameter type</prettyName>
-        <size>60</size>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
         <unmodifiable>0</unmodifiable>
-        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+        <values>Unknown|Wiki</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
       </type>
     </class>
     <property>
       <defaultValue/>
     </property>
     <property>
-      <description>Specifies the list of columns to display (e.g. "doc.name,doc.space"). If not specified then the default column list of used ("doc.name,doc.space,doc.date,doc.author")</description>
+      <description>Specifies the list of properties (e.g., "doc.name,doc.space"). If not specified then the default properties list of used ("doc.name,doc.space,doc.date,doc.author")</description>
```

Comment:
old typo still there it seems: "the default properties *is* used" no?

Reply:
Indeed! Fixing that.

Reply:
Done in https://github.com/xwiki/xwiki-platform/pull/2775/commits/c821d6f0da1de98c3818980ea310bed501a7a66d